myapp/spec/system/tasks_spec.rb

```diff
@@ -4,12 +4,19 @@
   let!(:task) { create(:task) }
 
   describe '#index' do
+    let!(:task_list) { create_list(:task, 4) }
+    let!(:task_order_by) { Task.order(created_at: :desc) }
```

Comment:
```suggestion
    let!(:tasks_order_by_created_at_desc) { Task.order(created_at: :desc) }
```
【任意】task_order_by_xxx ← 何でorder_byするのかを期待していたが、それがないとなんのorderになるのか少し分かりづらいですね。
あと複数形なので、tasksを使うのがおすすめです👍🏻

Reply:
確かにです、ありがとうございます修正します！